guides/locations/countries-and-states.md

```diff
@@ -0,0 +1,35 @@
+# Countries and states
+
+Solidus uses the list of countries and states provided by [Carmen][carmen-repo],
+which uses the countries and states available in the [Debian `iso-codes`
+package][debian-iso-codes]. Solidus creates new `Spree::Country` and
+`Spree::State` objects for each country and state.
+
+Carmen is generally up-to-date and should provide Solidus with any country or
+state you would ever need.
+
+[carmen-repo]: https://github.com/carmen-ruby/carmen
+[debian-iso-codes]: https://packages.debian.org/sid/all/iso-codes
+
+## Countries
+
+If a country or state you require is not recognized, you can add it directly
+from the Solidus admin (**Settings -> Locations -> Countries**) or from your
+Rails console:
+
+```
+Spree::Country.create!(iso_name:"NEW COUNTRY", name:"New Country", states_required: true)
+```
+
+## States
+
+A state is any sub-region of a country, whether that is a province, district, or
+territory. If the state you require is not recognized, you can add it directly
```

Comment:
For some countries, dividing them up into sub-territories makes no sense. For those cases, we can determine that the `states_required` field on the country is set to false. 

If we do that, then the subregions for that country are not chosen for addresses. 

This makes sense for many European countries and any small country.

Reply:
I added a note about this. :+1: